L1Trigger/VertexFinder/src/VertexFinder.cc

```diff
@@ -770,17 +770,21 @@ namespace l1tVertexFinder {
     };
 
     enum HistogramBitWidths {
-      kBinSize = 10,                    // Width of a single bin in z
-      kBinFixedSize = 7,                // Width of a single z0 bin in fixed point representation
-      kBinFixedMagSize = 4,             // Width (magnitude) of a single z0 bin in fixed point representation
+      kBinSize = 8,                     // Width of a single bin in z
+      kBinFixedSize = 8,                // Width of a single z0 bin in fixed point representation
+      kBinFixedMagSize = 5,             // Width (magnitude) of a single z0 bin in fixed point representation
```

Comment:
Again, I'm not sure I understand the need for the enumerated type. Is there some function of the `enum` that I am not seeing or remembering that could not be accomplished via some `const` or `constexpr`?

Reply:
in past versions of c++ it used to be the good way to define compile-time integer constants, but it's true that since c++11 using constexpr its better

Reply:
As above, this should probably be addressed simultaneously across all the GTT emulator (CMSSW) and firmware (HLS) code at once, following consensus. Keeping things uniform (or close to it) between CMSSW and HLS, which live in entirely different repos, makes maintenance easier.